src/core/js/views/adaptView.js

```diff
@@ -171,6 +172,11 @@ define([
                 return this.$el.removeClass('display-none');
             }
             this.$el.addClass('display-none');
+        },
+        
+        onIsCompleteChange:function(model){
+            var isComplete=model.get('_isComplete');
```

Comment:
Can you use the second argument passed to the function rather than a new variable please?